op-program/client/boot_test.go

```diff
@@ -0,0 +1,51 @@
+package client
+
+import (
+	"io"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBootstrapOracle(t *testing.T) {
+	r, w := io.Pipe()
+	br := NewBootstrapOracleReader(r)
+	bw := NewBootstrapOracleWriter(w)
+
+	bootInfo := BootInfo{
+		Rollup:             new(rollup.Config),
+		L2ChainConfig:      new(params.ChainConfig),
+		L1Head:             common.HexToHash("0xfffff"),
+		L2Head:             common.HexToHash("0xfffff"),
+		L2Claim:            common.HexToHash("0xfffff"),
```

Comment:
We should assign these different values so the test is actually confirming the specific value is preserved for each field and doesn't somehow mix up the fields.

Reply:
Good point. Done.